app/models/monthly_statistics/by_age_group.rb

```diff
@@ -207,15 +209,15 @@ def age_group_query
                 c.id, f.id, age_group
         )
         SELECT
-            raw_data.status[2],
+            raw_data.status,
```

Comment:
Just wondered why we need to change the sort order?

Reply:
So it just returns the state now i.e. 'pending conditions, rather than [2, 'pending_conditions]